tests/test_rabbitmq.py

```diff
@@ -89,6 +89,69 @@ def do_work():
     # I expect backoff time to have passed between sucesss and failure
     assert 800 <= success_time - failure_time <= 1200
 
+def test_rabbitmq_actors_retry_with_priority_elevation_on_failure(rabbitmq_broker, rabbitmq_worker):
+    max_reached_priority = 0
+    incoming_priorities = []
+
+    def get_current_message_priority(middleware):
+        options = next(iter(deadline[2].options for deadline in middleware.deadlines.values() if deadline and deadline[2]))
+        return options.get('priority')
+
+    @remoulade.actor(max_retries=13, increase_priority_on_retry=True, max_backoff=1)
+    def do_work():
+        nonlocal max_reached_priority
+        nonlocal incoming_priorities
+        incoming_priorities.append(
+            get_current_message_priority(next(iter(middleware for middleware in do_work.broker.middleware if hasattr(middleware, 'deadlines'))))
+        )
+        max_reached_priority += 1
```

Comment:
realy need max_reached_priority, I guess incoming_priorities is enouth, and also maybe only 4 retries is enough